lib/clients/rtm/client.js

```diff
@@ -47,8 +47,6 @@ var wsSocketFn = require('../transports/ws');
  * @param {boolean} opts.autoReconnect Whether or not to automatically reconnect when the connection
  *     closes.
  * @param {boolean} opts.useRtmConnect True to use rtm.connect rather than rtm.start
- * @param {number} opts.maxReconnectionAttempts
```

Comment:
good call, I think we can turn them into a retry policy. Important to note that the current retry behavior is to use `FIVE_RETRIES_IN_FIVE_MINUTES` _first_, and only once that has failed do we count it as a "connection attempt" and apply the backoff defined here. In other words, we're already way off-base in terms of applying these variables.